### PR TITLE
Update URLs to point to re-git.

### PR DESCRIPTION
**Pull request by KineticTheory**
### Background

I found a few more URLs in source code that were pointing to redmine.  I have updated these to point to re-git or I have removed the reference.

### Description of changes

* Touching `ApplicationUnitTest.cmake` forced me to implement the `cmake-format` fixes.

### Status

* Reference:
  * [Pre-Merge Code Review](https://github.com/lanl/Draco/wiki/Style-Guide)
  * [CDash Status](https://rtt.lanl.gov/cdash3/index.php?project=Draco&filtercount=1&showfilters=1&field1=buildname&compare1=63&value1=-pr)
* Checks
  * [x] Travis/Appveyor CI checks pass
  * [x] Code coverage does not decrease
  * [x] [Valgrind test passes](https://rtt.lanl.gov/cdash3/index.php?project=Draco)
  * [x] [Toss3 checks pass](https://rtt.lanl.gov/cdash3/index.php?project=Draco)
  * [x] [Trinitite checks pass](https://rtt.lanl.gov/cdash3/index.php?project=Draco)
  * [ ] Code reviewed/approved, sufficient DbC checks, testing, documentation

### CODE REVIEW

File: autodoc/README.autodoc [hunk 1]
```diff
@@ -1,16 +1,15 @@
...
-(https://www.stack.nl/~dimitri/doxygen). If available, this build can use
-graphviz (dot), dia and mscgen to generate graphics like dependency diagrams,
-UML and sequence diagrams.
+Developer documentation is generated using doxygen (https://www.stack.nl/~dimitri/doxygen). If
+available, this build can use graphviz (dot), dia and mscgen to generate graphics like dependency
+diagrams, UML and sequence diagrams.
 
 Documentation is generated nightly and published to the LANL Yellow Network at
-https://rtt.lanl.gov/autodoc.
+https://kellyt.re-pages.lanl.gov/jayenne/draco/index.html.
```
Comment (attom) [inline autodoc/README.autodoc]: This is the first I've heard of re-pages. Is it available to anyone?

File: CMakeLists.txt [hunk 2]
```diff
@@ -7,8 +7,8 @@
 #
 # Build notes:
 #
-# * https://rtt.lanl.gov/redmine/projects/draco/wiki/
-# * https://rtt.lanl.gov/redmine/projects/draco/wiki/Common_Configure_Options
+# * https://re-git.lanl.gov/draco/draco/-/wikis/home
+# * https://re-git.lanl.gov/draco/draco/-/wiki/Common_Configure_Options
```
Comment (attom) [inline CMakeLists.txt:11]: Do we want to use re-git or asc-git in our documentation?

File: autodoc/README.autodoc (hunk 1 above)
Comment (KineticTheory) [reply autodoc/README.autodoc]: Yes. This is a part of re-git.

File: CMakeLists.txt (hunk 2 above)
Comment (KineticTheory) [reply CMakeLists.txt:11]: For now, we need to use `re-git`.  `asc-git` is an alias but there are authentication issues related to forwarding requests from `asc-git` to `re-git`.

**Comment by KineticTheory** [comment]
@attom Can you merge this?  All tests passed.

**Pull request merged**